### DISCUSSION TITLE
native windows build exe

### DISCUSSION THREAD

**Discussion by tjumsl** [idaholab/moose#20287] [Q&A General] [answered]
I noted currently the instruction for windows installation is using conda and WSL, is there any way to get windows native builds (exe file) through cygwin or mingw?

Only the core server modulue is needed, post processing module is not necessary.

**Comment by GiudGiud** [comment]
Hello

No we have not developed a windows native build. We are interested in having this though, if anyone wants to tackle it

Guillaume

**Comment by dschwen** [comment, marked as answer]
We have tried our hand on a native Windows build before. This effort might get revived as there is a libmesh contributor who's working on native support for VC.
